pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotTableIdealStateBuilder.java

```diff
@@ -99,4 +99,28 @@ public static List<PartitionGroupMetadata> getPartitionGroupMetadataList(StreamC
       throw new RuntimeException(fetcherException);
     }
   }
+
+  /**
+   * Fetches the list of {@link PartitionGroupMetadata} for the new partition groups for the stream,
+   * with the help of the {@link PartitionGroupConsumptionStatus} of the current partitionGroups.
+   * In particular, this method is used to fetch from multiple stream topics.
+   * @param streamConfigs
+   * @param partitionGroupConsumptionStatusList
+   * @return
+   */
+  public static List<PartitionGroupMetadata> getPartitionGroupMetadataList(List<StreamConfig> streamConfigs,
```

Comment:
Deprecate the old method or remove it. Please also clean up all usages of the old one